script.commands/changelog.txt

```diff
@@ -0,0 +1,9 @@
+1.0.0
```

Comment:
Changelog is missing your new version. Also, Kodi now has the `<news>` element in addon xml to display the news regarding the latest addon version. Consider to use it.